lua-http-parser.c

```diff
@@ -398,7 +460,11 @@ static int lhp_execute(lua_State* L) {
     parser->data = NULL;
 
     /* replace nil place-holder with 'result' code. */
+#if LUA_VERSION_NUM > 502
+    lua_pushinteger(L, result);
```

Comment:
Can't we just always use lua_pushinteger()? That exists even in Lua 5.1:

https://www.lua.org/manual/5.1/manual.html#lua_pushinteger

Reply:
lua_Integer on Windows x86 is 32 bit but size_t is int64_t.
So I prefer use double in that case

Reply:
This is a really good point... although isn't it unsigned? Maybe we should always lua_pushnumber()?

Reply:
With double we can handle 2^53 on Lua 5.1-5.2
With integer on Lua 5.3 we can handle 2^63 unsigned values.
So i think it is enouth in this case
I only afraid obout chunked_header.
Request can have any value in this field.
I doubt that anyone will send such huge data in one chunk,
but send big number in header is possible.
I think we can stop parse data if we got such value in header.
Not sure how to stop parser with specific error by hand.

Reply:
I like the change.  As long as Lua code using this module on Lua 5.1 and Lua 5.3 will work the same without changes.